maven/bnd-maven-plugin/src/main/java/aQute/bnd/maven/plugin/BndMavenTestsPlugin.java

```diff
@@ -0,0 +1,118 @@
+package aQute.bnd.maven.plugin;
+
+/*
+ * Copyright (c) Paremus and others (2015, 2016). All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import java.io.File;
+import java.util.List;
+
+import org.apache.maven.model.Resource;
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.plugins.annotations.ResolutionScope;
+
+import aQute.bnd.osgi.Builder;
+import aQute.bnd.osgi.Constants;
+
+@Mojo(name = "bnd-process-tests", defaultPhase = LifecyclePhase.PROCESS_TEST_CLASSES, requiresDependencyResolution = ResolutionScope.COMPILE, threadSafe = true)
+public class BndMavenTestsPlugin extends AbstractBndMavenPlugin {
+
+	/**
+	 * If true, make the tests artifact of fragment of the main artifact.
+	 */
+	@Parameter(defaultValue = "false")
+	private boolean									fragment;
+
+	/**
+	 * Possible values are {@link TestCases#junit3 junit3},
+	 * {@link TestCases#junit4 junit4}, {@link TestCases#junit5 junit5} and
+	 * {@link TestCases#all all}
+	 */
+	@Parameter(defaultValue = "junit4")
```

Comment:
I would think junit5 would be the default these days...

Reply:
we don't even have ability to run junit5 yet...

Reply:
but I'll make it the default...